cmd/lakectl/cmd/root.go

```diff
@@ -94,6 +94,10 @@ type Configuration struct {
 		// setting FixSparkPlaceholder to true will change spark placeholder with the actual location. for more information see https://github.com/treeverse/lakeFS/issues/2213
 		FixSparkPlaceholder bool `mapstructure:"fix_spark_placeholder"`
 	}
+	Local struct {
+		// SkipIrregularFiles - By default lakectl local fails if local directory contains a symbolic link. When set, lakectl will ignore the symbolic links instead.
+		SkipIrregularFiles bool `mapstructure:"skip_irregular_files"`
```

Comment:
I would prefer `skip_not_regular_files`.  I cannot find any references to "irregular files", even we say "not a regular file" in the error message.

Or reverse the flag and give it a positive description and default `true` with `fail_on_non_regular_file`.